### PR TITLE
bumping dependencies

### PR DESCRIPTION
**Pull request by thekp**
Resolves n/a

**Overall change:** bump packages at top level
**todo: run npm install to update package-lock**

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

**Review by dr3 — APPROVED** [review]

**Review by jamesbhobbs — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -54,22 +54,22 @@
...
     "@babel/preset-env": "^7.5.5",
     "@babel/preset-react": "^7.0.0",
-    "@bbc/gel-foundations": "^3.2.0",
-    "@bbc/psammead-assets": "^2.0.0",
-    "@bbc/psammead-brand": "^4.2.3",
-    "@bbc/psammead-caption": "^2.1.2",
-    "@bbc/psammead-copyright": "^1.1.2",
+    "@bbc/gel-foundations": "^3.2.1",
+    "@bbc/psammead-assets": "^2.0.1",
+    "@bbc/psammead-brand": "^4.2.4",
```
Comment (sareh) [inline package.json]:
```suggestion
    "@bbc/psammead-brand": "^4.2.5",
```

File: package.json [hunk 2]
```diff
@@ -54,22 +54,22 @@
...
+    "@bbc/psammead-caption": "^2.1.3",
```
Comment (sareh) [inline package.json]:
```suggestion
    "@bbc/psammead-caption": "^2.1.4",
```

File: package.json [hunk 3]
```diff
@@ -54,22 +54,22 @@
...
+    "@bbc/psammead-copyright": "^1.1.3",
```
Comment (sareh) [inline package.json]:
```suggestion
    "@bbc/psammead-copyright": "^1.1.4",
```

File: package.json [hunk 4]
```diff
@@ -54,22 +54,22 @@
...
     "@bbc/psammead-image": "^1.1.2",
```
Comment (sareh) [inline package.json]:
```suggestion
    "@bbc/psammead-image": "^1.1.3",
```

File: package.json [hunk 5]
```diff
@@ -54,22 +54,22 @@
...
-    "@bbc/psammead-inline-link": "^1.2.2",
-    "@bbc/psammead-locales": "^1.2.1",
-    "@bbc/psammead-media-indicator": "^2.3.2",
-    "@bbc/psammead-paragraph": "^2.1.2",
-    "@bbc/psammead-story-promo": "2.3.1",
-    "@bbc/psammead-storybook-helpers": "^3.3.2",
-    "@bbc/psammead-test-helpers": "^1.1.0",
-    "@bbc/psammead-timestamp": "^2.1.2",
-    "@bbc/psammead-visually-hidden-text": "^1.1.1",
+    "@bbc/psammead-image-placeholder": "^1.1.3",
```
Comment (sareh) [inline package.json]:
```suggestion
    "@bbc/psammead-image-placeholder": "^1.1.4",
```

File: package.json [hunk 6]
```diff
@@ -54,22 +54,22 @@
...
+    "@bbc/psammead-inline-link": "^1.2.3",
```
Comment (sareh) [inline package.json]:
```suggestion
    "@bbc/psammead-inline-link": "^1.2.4",
```

File: package.json [hunk 7]
```diff
@@ -54,22 +54,22 @@
...
+    "@bbc/psammead-locales": "^1.2.2",
+    "@bbc/psammead-media-indicator": "^2.3.3",
```
Comment (sareh) [inline package.json]:
```suggestion
    "@bbc/psammead-media-indicator": "^2.4.1",
```

File: package.json [hunk 8]
```diff
@@ -54,22 +54,22 @@
...
+    "@bbc/psammead-paragraph": "^2.1.3",
```
Comment (sareh) [inline package.json]:
```suggestion
    "@bbc/psammead-paragraph": "^2.1.4",
```

File: package.json [hunk 9]
```diff
@@ -54,22 +54,22 @@
...
+    "@bbc/psammead-story-promo": "2.3.2",
```
Comment (sareh) [inline package.json]:
```suggestion
    "@bbc/psammead-story-promo": "2.3.3",
```

File: package.json [hunk 10]
```diff
@@ -54,22 +54,22 @@
...
+    "@bbc/psammead-storybook-helpers": "^3.3.3",
```
Comment (sareh) [inline package.json]:
```suggestion
    "@bbc/psammead-storybook-helpers": "^4.0.0",
```

File: package.json [hunk 11]
```diff
@@ -54,22 +54,22 @@
...
+    "@bbc/psammead-test-helpers": "^1.1.1",
+    "@bbc/psammead-timestamp": "^2.1.3",
```
Comment (sareh) [inline package.json]:
```suggestion
    "@bbc/psammead-timestamp": "^2.1.4",
```

File: package.json [hunk 12]
```diff
@@ -43,7 +43,7 @@
   },
   "homepage": "https://github.com/bbc/psammead#readme",
   "dependencies": {
-    "@bbc/psammead-styles": "^2.0.1",
+    "@bbc/psammead-styles": "^2.0.2",
```
Comment (sareh) [inline package.json]: Please move this to devDependencies.

**Review by sareh — DISMISSED** [review]
A lot of dependencies have been updated since this PR was created. :) Could you please update these versions.

**Pull request merged**